.github/scripts/validateActionsAndWorkflows.sh

```diff
@@ -32,20 +47,17 @@ done
 
 # Wait for the background builds to finish
 for PID in ${ASYNC_PROCESSES[*]}; do
-  if ! wait $PID; then
-    EXIT_CODE=$?
+  wait $PID
+  RESULT=$?
+  if [[ $RESULT != 0 ]]; then
+    EXIT_CODE=$RESULT
   fi
 done
```

Comment:
I think this is what fixed the bug, right?

Reply:
Yeah. When you have an `if` statement that in-and-of-itself is a command and so it changes the value of `$?`.